.github/workflows/build_and_test.yaml

```diff
@@ -0,0 +1,67 @@
+# See also https://github.com/EVerest/everest-core/tree/main/.github/workflows
+# TODO: modify to reuse the above workflow to DRY up CI.
+
+name: Build and test libtimer
+on: 
+  pull_request: {}
+  push:
```

Comment:
```suggestion
```
> https://github.com/US-JOET/libtimer/actions/runs/7821295738 - passing within our forked repo.

Good, could you then remove `push` again, please. It should be only temporary there

Reply:
I believe that during a conversation in the CI/Testing working group there was a desire to run the checks on branch and PRs. This would allow for branch runs. Removed until this can be confirmed.